JackyZhang1219/books-and-breakfast-app
# Harry

### Description
Please explain the changes you made here.

Added App Navigation using React Navigation and enhanced Home Screen UI/UX. GroupMe button added as well as a function that can prompt the opening of other apps. 

### QA
Please explain in detailed steps how the reviewer can test your changes and what the expected behaivors are (visual would help!). Consider different test cases/scenarios. 

- Clicking on 'Navigation' or 'Tracker' buttons should open a separate page.
- (should be able to go back to Home through the default headers at the top)
- Clicking on 'Link to GroupMe' should open the GroupMe app with the invite ready.

Someone help me with the typing for React Navigation's navigation() function because somehow it doesn't work right now and I've tried other alternatives that didn't work... :(

### Packages
Please describe any additional packages you added to package.json.

Both for navigation.

"@react-navigation/native": "^6.1.17"
"@react-navigation/stack": "^6.3.29"

### Checklist
